src/rgw/rgw_usage.cc

```diff
@@ -146,6 +146,37 @@ int RGWUsage::show(const DoutPrefixProvider *dpp, rgw::sal::Store* store,
     formatter->close_section(); // summary
   }
 
+  ret = rgw_user_sync_all_stats(dpp, store, user, null_yield);
```

Comment:
Add comment on explaining why you have added code here & also what all things you are doing here.

Reply:
Addressed